easybuild/easyconfigs/c/CondaCreate/requirements.txt

```diff
@@ -0,0 +1 @@
+perl-app-cpanminus
```

Comment:
@jerowe you can update the easyblock to just skip the extract step, by adding:

```python
def extract_step(self):
    """No sources provided, so nothing to unpack."""
    pass
```

I'm a bit weary of this though, since that means this will require you to be online all the time.

Also, shouldn't we enforce people to specify the packages to install with a specific version?